api_specifications/application-api-0.1.0.yaml

```
@@ -0,0 +1,411 @@
+          maxLength: 48
+          description: number assigned to the application. applications are numbered serially for the court and affixed with the year.
+          example: Application 1 of the year 2024
+        applicationCMPNumber:
+          type: string
+          description: number assigned to the application when status is pending approval
+          example: CMP/1/2024          
+        issuedBy:
+          $ref: '#/components/schemas/IssuedBy'
+        reasonForApplication:
+          type: string
+          description : Reason For Application eg.- Reason for Rescheduling, Reason for Extension
```

YAMLlint (1.37.1)

[error] too many spaces before colon

(colons)